media/img/frown.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns='http://www.w3.org/2000/svg' width='24' height='24' viewBox='0 0 16 16'><circle fill='#fc0' cx='8' cy='8' r='7'/><path fill='#424242' d='M10.973 6.5c0 .5-.447.5-1 .5s-1 0-1-.5c0-1.104 1-1.5 1-1.5s1 .396 1 1.5zM6.973 6.5c0 .5-.447.5-1 .5s-1 0-1-.5c0-1.104 1-1.5 1-1.5s1 .396 1 1.5zM5.104 12c.459-1.146 1.593-2 2.869-2 1.277 0 2.439.854 2.912 2h.916c-.475-1.678-1.998-2.917-3.828-2.917S4.618 10.322 4.143 12h.961z'/></svg>
```

Comment:
Where did this svg file originate from? did you draw it?

Reply:
All the svgs are from the vscode feedback dialog. These files are actually unnecessary since I chose to inline them in the css.